### ISSUE TITLE
Healthmonitor resurrects on SSH login alerts

### ISSUE THREAD

**Issue by voelzmo** [cloudfoundry/bosh#1658]
Whenever a user does ssh into a VM (manual or by doing `bosh ssh`), the agents sends an alert to the Healthmonitor (https://github.com/cloudfoundry/bosh-agent/blob/master/agent/alert/ssh_adapter.go#L16). The HM has a pretty simple check to figure out if resurrection needs to be triggered for an alert, which just checks for the presence of a few properties (https://github.com/cloudfoundry/bosh/blob/master/src/bosh-monitor/lib/bosh/monitor/plugins/resurrector.rb#L38).

Turns out, those ssh alerts from the agent do have those properties set, here is an example what it looks like:
```
<Bosh::Monitor::Events::Alert:0x00557d12802a98 @attributes={"id"=>"4427b8ec-f0c2-483d-5f9f-328f4b340a8f", "severity"=>4, "title"=>"SSH Login", "summary"=>"Accepted publickey for vcap from <IP> port 33080 ssh2: RSA e3:5f:58:81:63:dc:86:04:c4:44:ec:64:fa:e2:6a:5c", "created_at"=>1493049382, "source"=>"service-fabrik-0116-dc8e0796-f55b-41a1-aee4-5403e1756f3b: postgresql_slave_z1(2fba6f93-a142-49d9-845a-2a131f8a4f85) [id=500e3447-a4cb-4ef3-b5ea-aecd648f7e4f, index=0, cid=i-0024e3bb42d1260e3]", "deployment"=>"service-fabrik-0116-dc8e0796-f55b-41a1-aee4-5403e1756f3b", "job"=>"postgresql_slave_z1", "instance_id"=>"2fba6f93-a142-49d9-845a-2a131f8a4f85"}, @kind=:alert, @logger=<Logging::Logger:0x2abe892b9928 name="/var/vcap/sys/log/health_monitor/health_monitor.log">, @errors=#<Set: {}>, @id="4427b8ec-f0c2-483d-5f9f-328f4b340a8f", @severity=4, @title="SSH Login", @summary="Accepted publickey for vcap from <IP> port 33080 ssh2: RSA e3:5f:58:81:63:dc:86:04:c4:44:ec:64:fa:e2:6a:5c", @source="service-fabrik-0116-dc8e0796-f55b-41a1-aee4-5403e1756f3b: postgresql_slave_z1(2fba6f93-a142-49d9-845a-2a131f8a4f85) [id=500e3447-a4cb-4ef3-b5ea-aecd648f7e4f, index=0, cid=i-0024e3bb42d1260e3]", @deployment="service-fabrik-0116-dc8e0796-f55b-41a1-aee4-5403e1756f3b", @job="postgresql_slave_z1", @instance_id="2fba6f93-a142-49d9-845a-2a131f8a4f85", @created_at=2017-04-24 15:56:22 UTC>
```

Director doesn't do anything, but still executes a scan_and_fix task.

**Comment by voelzmo** [comment]
Updated to reflect that this also happens when doing `bosh ssh`

**Comment by geofffranks** [comment]
I'm also seeing this behavior due to the following other messages:

```
health_monitor.log-I, [2017-04-27T16:11:58.438951 #52726]  INFO : [ALERT] Alert @ 2017-04-27 16:11:58 UTC, severity 1: process is not running
health_monitor.log-I, [2017-04-27T16:11:58.439098 #52726]  INFO : (Event logger) notifying director about event: Alert @ 2017-04-27 16:11:58 UTC, severity 1: process is not running
health_monitor.log:W, [2017-04-27T16:11:58.439553 #52726]  WARN : (Resurrector) notifying director to recreate unresponsive VM: warden-prometheus prometheus/05d41d5e-0afc-4894-8e0f-45b4c8260d0f
```
(it appears a process is repeatedly crashing on that VM due to lack of disk space, but monit keeps restarting it.. yet this is trying to trigger a VM recreation?)

```
I, [2017-04-27T16:06:20.072576 #8017]  INFO : [ALERT] Alert @ 2017-04-27 16:06:20 UTC, severity 4: Connection closed by 101.200.122.213 [preauth]
I, [2017-04-27T16:06:20.072745 #8017]  INFO : (Event logger) notifying director about event: Alert @ 2017-04-27 16:06:20 UTC, severity 4: Connection closed by 101.200.122.213 [preauth]
W, [2017-04-27T16:06:20.073258 #8017]  WARN : (Resurrector) notifying director to recreate unresponsive VM: eastus-jumpbox jumpbox/5b46a0c7-1724-438e-a560-eb4a94044672
```
(an artifact of having a public IP)

**Comment by voelzmo** [comment]
Validated this is fixed in 3586.x, tested on 3586.36.
